heroic-elasticsearch-utils/src/main/java/com/spotify/heroic/elasticsearch/NodeClientSetup.java

```diff
@@ -49,6 +50,8 @@ public Client setup() throws Exception {
             .put("node.name", InetAddress.getLocalHost().getHostName())
             .put("discovery.zen.ping.multicast.enabled", false)
             .putArray("discovery.zen.ping.unicast.hosts", seeds)
+            // Fixing path.home not configured error in unit test
+            .put("path.home", "/tmp/" + UUID.randomUUID())
```

Comment:
Make use of [`Files.createTempDirectory()`](http://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempDirectory(java.lang.String,%20java.nio.file.attribute.FileAttribute...)) since it does the right thing already.